packages/shared/providers.yaml

```diff
@@ -3700,6 +3700,31 @@ microsoft-power-bi:
         base_url: https://api.powerbi.com
     docs: https://docs.nango.dev/integrations/all/microsoft-power-bi
 
+mindbody:
+    display_name: Mindbody
+    categories:
+        - fitness
```

Comment:
Please change this to productivity as you had included above.

Reply:
changed